layers/+email/mu4e/packages.el

```diff
@@ -36,6 +38,9 @@
       (add-to-list 'mu4e-view-actions
                    '("View in browser" . mu4e-action-view-in-browser) t)
 
+      (when (configuration-layer/layer-usedp 'org)
+        (require 'org-mu4e))
```

Comment:
Not unless org mode is always guaranteed to be loaded, which I don't think is the case. org-mu4e adds functionality to both mu4e and org mode. Changing `(require 'org-mu4e)` to `(require 'org)` would work b/c of the `(with-eval-after-load` but that seems a bit convoluted. :)